### PR TITLE
Expected output for smaller init tearingsets

### PR DESCRIPTION
**Pull request by ptaeuber**
This is the expected Output for OpenModelica/OMCompiler#785, which leads to smaller initialization tearing sets.

There are still three failing models:
`./openmodelica/cppruntime/genDebugSymbols/Modelica.Fluid.Examples.BranchingDynamicPipes_genDebugSymbols.mos
        ./openmodelica/cppruntime/hardcodedStartValues/Modelica.Fluid.Examples.BranchingDynamicPipes_hardcodedStartValues.mos
        ./simulation/libraries/3rdParty/ThermoPower/ThermoPower.Test.DistributedParameterComponents.TestFlow1D2phChen.mos
`

### CODE REVIEW

File: openmodelica/fmi/ModelExchange/1.0/JuliansBib.mos [hunk 1]
```diff
@@ -80,10 +80,10 @@ val(s, 1);
...
 //     messages = ""
 // end SimulationResult;
-// 127.2093118472222
-// -1.473681007759564
-// 12720.93118472221
-// -2.52587518528955
+// 29.87032083226751
+// -2.843354040235435
+// 2987.032083226751
+// -1.180467185324341
```
Comment (lochel) [inline openmodelica/fmi/ModelExchange/1.0/JuliansBib.mos]: These results need to be verified.

File: simulation/modelica/tearing/Tearing16-celMC3.mos [hunk 2]
```diff
@@ -78,9 +78,9 @@ val(i3,0.0); getErrorString();
 // ""
 // 0.0
 // ""
-// -12.56637061435917
+// 3.302928029719854e-13
 // ""
-// -12.56637061435917
+// 3.302928029719854e-13
```
Comment (lochel) [inline simulation/modelica/tearing/Tearing16-celMC3.mos]: These results need to be verified.

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2016-09-16_17-31-45](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3136/).

File: simulation/modelica/tearing/Tearing16-celMC3.mos (hunk 2 above)
Comment (ptaeuber) [inline simulation/modelica/tearing/Tearing16-celMC3.mos]: As this is the trivial solution it's definitely correct.

File: openmodelica/fmi/ModelExchange/1.0/JuliansBib.mos (hunk 1 above)
Comment (ptaeuber) [inline openmodelica/fmi/ModelExchange/1.0/JuliansBib.mos]: It is correct, other tools produce the same trajectories with those start values.
The difference in FMU and normal simulation result is due to different nonlinear solvers.

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2016-09-27_16-53-12](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3161/console).

**Pull request closed without merging**